Directory.Build.props

```diff
@@ -4,11 +4,16 @@
     <LangVersion>latest</LangVersion>
     <PackageProjectUrl>https://github.com/graphql-dotnet/server</PackageProjectUrl>
     <RepositoryUrl>https://github.com/graphql-dotnet/server</RepositoryUrl>
-    <RepositoryType>Git</RepositoryType>
+    <RepositoryType>git</RepositoryType>
     <Copyright>Pekka Heikura</Copyright>
     <Authors>Pekka Heikura</Authors>
     <Product>graphql-dotnet server</Product>
     <Company>graphql-dotnet</Company>
+    <VersionPrefix>3.5.0</VersionPrefix>
+    <VersionSuffix>alpha</VersionSuffix>
+    <AssemblyName>GraphQL.Server.$(MSBuildProjectName)</AssemblyName>
```

Comment:
This will allow to unify naming and in case of new packages not to make mistakes with naming conventions.